lib/streamlit/elements/lib/options_selector_utils.py

```diff
@@ -251,3 +251,19 @@ def maybe_coerce_enum_sequence(register_widget_result, options, opt_sequence):
         ),
         register_widget_result.value_changed,
     )
+
+
+def create_mappings(
```

Comment:
nit: maybe add a docstring comment on what the function is doing.

Reply:
Yeah great point, I have added a doc string!